<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings_Dialog</class>
 <widget class="QDialog" name="Settings_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>421</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_settings">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Serial Settings</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_DMM">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>421</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Digial Multimeter (DMM) configuration</string>
   </property>
   <widget class="QLabel" name="label_DMM_port">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DMM Port:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_DMM_port">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>23</y>
      <width>311</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_DMM_baudrate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Baudrate:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_DMM_databits">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DataBits:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_DMM_stopbits">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>StopBits:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_DMM_parity">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Parity:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_DMM_baudrate">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>53</y>
      <width>311</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_DMM_databits">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>83</y>
      <width>311</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_DMM_stopbits">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>113</y>
      <width>311</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_DMM_parity">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>143</y>
      <width>311</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_DMM_flowcontrol">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FlowControl:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_DMM_flowcontrol">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>173</y>
      <width>311</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_DMM_Test_Connection">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Test Connection</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Rescan">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Re-scan ports</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Apply">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>290</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Settings_Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Settings_Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
